lib/bundler/compact_index_client/cache.rb

```diff
@@ -92,19 +94,9 @@ def lines(path)
         header ? lines[header + 1..-1] : lines
       end
 
-      def parse_gem(string)
-        version_and_platform, rest = string.split(" ", 2)
-        version, platform = version_and_platform.split("-", 2)
-        dependencies, requirements = rest.split("|", 2).map {|s| s.split(",") } if rest
-        dependencies = dependencies ? dependencies.map {|d| parse_dependency(d) } : []
-        requirements = requirements ? requirements.map {|r| parse_dependency(r) } : []
-        [version, platform, dependencies, requirements]
-      end
-
-      def parse_dependency(string)
-        dependency = string.split(":")
-        dependency[-1] = dependency[-1].split("&") if dependency.size > 1
-        dependency
+      def parse_gem(line)
+        @dependency_parser ||= GemParser.new
```

Comment:
Is it intentional to call the instance variable `@dependency_parser` instead of `@gem_parser`?

Reply:
Oh, sorry. It's my fault.
I'll fix this.

At first I was implementing `GemParser` as `DependencyParser` then I renamed it to improve naming.
But I forgot to change this instance variable name.